src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -98,14 +97,14 @@ public Lift() {
         addChild(liftLidar);
         addChild(findLiftHeight());
 
-        Preferences.getInstance().getDouble(RobotMap.Preferences.K_P, 1);
-        Preferences.getInstance().getDouble(RobotMap.Preferences.K_I, 1);
-        Preferences.getInstance().getDouble(RobotMap.Preferences.K_D, 1);
-        Preferences.getInstance().getDouble(RobotMap.Preferences.K_F, 1);
-        Preferences.getInstance().getDouble(RobotMap.Preferences.LIFT_UP_DOWN_INCREMENT, 1);
-        Preferences.getInstance().getDouble(RobotMap.Preferences.UP_MAX_SPEED, 1);
-        Preferences.getInstance().getDouble(RobotMap.Preferences.DOWN_MAX_SPEED, 1);
-        Preferences.getInstance().getBoolean(RobotMap.Preferences.USE_LIDAR, false);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_P, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_I, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_D, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.K_F, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.LIFT_UP_DOWN_INCREMENT, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.UP_MAX_SPEED, 1);
+        PreferenceStrings.setDefaultDouble(PreferenceStrings.DOWN_MAX_SPEED, 1);
+        Preferences.getInstance().putBoolean(PreferenceStrings.USE_LIDAR, false);
```

Comment:
Looks like you didn't get back to change this one.

Reply:
that needs `setDefaultBoolean`